src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -767,6 +768,11 @@ function reenableFieldsForSubmission() {
     $(':disabled').prop('disabled', false);
 }
 
+function disableSubmissionButton() {
```

Comment:
I personally don't think 'SLAP' is very applicable to this function.

Two different suggestions
1. `addLoadingIndicator($('#response_submit_button'), '')` could be a single line with a comment.
2. move this function closer to its counterpart. 700 lines separation is a little excessive.